app/javascript/packs/application.css

```diff
@@ -0,0 +1,9 @@
+/* 
+Any CSS added to this file or imported from this file, e.g. `@import '../stylesheets/my-css.css'`,

```

Comment:
We now have two `application.css` - one in `app/javascript/` and one in `app/assets/stylesheets/`. Do we need both?

Reply:
Done